tests/unit/PluginTest.php

```
@@ -0,0 +1,75 @@
+ *
+ * @author Wilmer Arambula <terabytesoftw@gmail.com>
+ * @since 0.1
+ */
+#[Group('scaffold')]
+final class PluginTest extends TestCase
+{
+    public function testEventSubscriberRegistersPostCreateProjectCmd(): void
+    {
+        self::assertArrayHasKey(
+            ScriptEvents::POST_CREATE_PROJECT_CMD,
+            EventSubscriber::getSubscribedEvents(),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\scaffold\EventSubscriber' in method 'testEventSubscriberRegistersPostCreateProjectCmd'. (undefined)

(StaticAccess)